tests/smoke/events.js

```diff
@@ -155,6 +155,47 @@
 					observer.assertRootIs(inner);
 					observer.assert(["spellCheckAbort"]);
 				}
+		},
+		'test spellcheck can batch ajax calls for a document with multiple blocks': function() {
+			var bot = this.editorBot,
+				editor = bot.editor,
+				resumeAfter = bender.tools.resumeAfter,
+				observer = observeSpellCheckEvents(editor),
+				fiveBlockHtml = '<p>This</p><p>is</p><ul><li>five</li><li>block</li></ul><p>test</p>',
+				sixBlockHtml = '<p>This</p><p>is</p><p>a</p><ul><li>six</li><li>block</li></ul><p>test</p>';
+
+				// this test assumes that the BLOCK_REQUEST_LIMIT is set to the default
+				// of 5 blocks per AJAX request
```

Comment:
You don't have to assume, :) you can actually go to the top of this file and add the config (same object where autostart is set to false)